app/src/main/java/com/example/paritosh/stormy/WeatherContract.java

```diff
@@ -5,13 +5,18 @@
 import android.support.annotation.StringRes;
 
 import com.example.paritosh.stormy.model.CurrentWeatherDataBindingModel;
+import com.example.paritosh.stormy.model.Hourly;
+
+import java.util.ArrayList;
 
 public interface WeatherContract {
 
     interface WeatherView {
 
         void render(CurrentWeatherDataBindingModel model);
 
+        void loadHourlyData(ArrayList<Hourly> hourlyData);
```

Comment:
Two things:
1. Name should be updateHourlyData
2. You shouldn't pass two models separately to the view through two different methods. Try to include hourlyData within CurrentWeatherDataBindingModel and delete this method. Collect the hourlyData in the render implementation in WeatherActivity.